code/modules/surgery/organs/brain.dm

```diff
@@ -130,18 +132,29 @@
 		owner.setBrainLoss(120)
 
 /obj/item/organ/internal/brain/on_life()
-	if(decoy_brain || damage < 10)
+	if(decoy_brain)
 		return
-	switch(damage)
-		if(10 to 30)
+	if(ismonkeybasic(owner))
```

Comment:
This feels like an unrelated change, why is this here?

Reply:
It's supposed to be for testing, whoops